libr/core/cmd_print.c

```diff
@@ -3284,13 +3297,22 @@ static int cmd_print_blocks(RCore *core, const char *input) {
 			r_cons_println (pj_string (pj));
 			pj_free (pj);
 			break;
-		case 'h':
-			// r_cons_printf ("  total    | flags funcs cmts syms str  |\n");
-			r_cons_printf ("|-------------)----------------------------|\n");
-			r_cons_printf ("|    total    | %4d %4d %4d %4d %4d   |\n",
-						   total.flags, total.functions, total.comments, total.symbols, total.strings);
-			r_cons_printf ("`-------------'----------------------------'\n");
+		case 'h': {
+			char *flags = r_str_newf ("%d", total.flags);
+			char *funcs = r_str_newf ("%d", total.functions);
+			char *comments = r_str_newf ("%d", total.comments);
+			char *symbols = r_str_newf ("%d", total.symbols);
+			char *strings = r_str_newf ("%d", total.strings);
+			r_table_add_row (t, "total", flags, funcs, comments, symbols, strings, NULL);
```

Comment:
you can use sdb_fmt to avoid having to allocate and free a lot of strings, also, we can probably add an add_row_nums() to accept numbers instead of strings to avoid that ducktyping

Reply:
^